lib/ueberauth/strategy/cognito.ex

```diff
@@ -138,7 +180,7 @@ defmodule Ueberauth.Strategy.Cognito do
 
     expires_at =
       if token["expires_in"] do
-        System.system_time(:seconds) + token["expires_in"]
+        System.system_time(:second) + token["expires_in"]
```

Comment:
I'm not opposed, but just curious why you changed all the `:seconds` to `:second`. Just aesthetics?

Reply:
Apparently `:seconds` is deprecated in favor of `:second` now, I was getting a lot of noisy warnings.

Reply:
Ah, good to know. Thanks. 👍